src/test/java/tests/AddRemoveTest.java

```diff
@@ -0,0 +1,39 @@
+package tests;
+
+import io.qameta.allure.Description;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+public class AddRemoveTest extends BaseTest{
+        private static final Logger logger = LoggerFactory.getLogger(AddRemoveTest.class);
+
+        @Test
+        @Description("Verify user can click add and remove button")
+        public void clickAdd() throws InterruptedException, IOException {
```

Comment:
For test methods name use the name that tells what this test method is about or what this test case does. Example: instead of using clickAdd(), refactor it to something like verifyAddRemoveActionTest